src/components/radar/RadarAnimation.js

```diff
@@ -76,8 +78,22 @@ class RadarAnimation extends React.Component<Props> {
         position: MIDDLE_RIGHT,
     };
 
+    tetherRef = React.createRef<{ position: () => {} }>();
+
     radarAnimationID = uniqueId('radarAnimation');
 
+    isShown = () => {
+        const { isShown = true } = this.props;
```

Comment:
use `defaultProps`, this doesn't need to be a function if its a just returning a prop

Reply:
its defaulting the value if the prop isnt provided. id prefer not using defaultProps since it makes it less clear which props are optional.

Reply:
Not sure if I agree that defaultProps makes it less clear. Its a react standard and the above flow types explain that is is optional

Reply:
yeah, the fact that it needs a comment is kind of the issue... otherwise callers are forced to compare the Props and the defaultProps in order to determine which are optional and which aren't. 

seems like its being done here already though, so im fine using it if there is a strong feeling towards it.

Reply:
I guess its a matter of opinion but personally I don't agree the way you have it is clearer. The consumer would have to look at the component in the `isShown` to determine what the default value is vs looking at the `defaultProps`. I would prefer to stick to the existing pattern of using `defaultProps`

Reply:
done